### DISCUSSION TITLE
Difficulty Adjust mod is off, but circles are still affected. Even after relaunch

### DISCUSSION THREAD

**Discussion by rvnexe** [ppy/osu#23641] [Q&A]
### Type

Game behaviour

### Bug description

Recently I used difficulty adjust to enlarge circles for [HEALTH - Major Crimes [Normal] (by Ishkah) ](https://osu.ppy.sh/beatmapsets/1665172#osu/3399427)
After some plays, I turned it off, but circles are still large. On all beatmaps, even super-hard small circle ones.
I relaunched the game several times, but the circles are still large.

### Screenshots or videos

![osu_2023-05-23_18-08-22-1](https://github.com/ppy/osu/assets/89677246/3f8f5f59-810b-44f9-bd9d-9503eb22ecf0)
![osu_2023-05-23_18-15-37](https://github.com/ppy/osu/assets/89677246/a4e26c55-dec1-419b-85f6-fbb662a82f51)

### Version

2023.513.0-lazer

### Logs

[performance.log](https://github.com/ppy/osu/files/11545406/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11545407/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11545408/updater.log)
[database.log](https://github.com/ppy/osu/files/11545409/database.log)
[input.log](https://github.com/ppy/osu/files/11545410/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/11545412/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/11545413/network.log)

**Comment by bdach** [comment]
I cannot reproduce this issue. Please provide video evidence.

> I relaunched the game several times, but the circles are still large.

I would like to think that this is literally impossible.

**Reply by rvnexe** [reply]
I didn't recorded any video. It's something like Difficulty Adjust is on, but in the mods menu its off
